### PR TITLE
[ROOT-8839] Disabling LLVM_ENABLE_ZLIB option for LLVM

### PR DESCRIPTION
**Pull request by oshadura**
Updated PR for https://github.com/root-project/root/pull/1149

### CODE REVIEW

**Comment by oshadura** [comment]
@vgvassilev I created a new PR, can you take a look please?

**Comment by oshadura** [comment]
screenshot from 2017-10-11 13-57-56
reported issue seems to be fixed..

File: interpreter/CMakeLists.txt [hunk 1]
```diff
@@ -20,6 +20,7 @@ if(cxx17)
 elseif(cxx14)
   set(LLVM_ENABLE_CXX1Y ON CACHE BOOL "" FORCE)
 endif()
+set(LLVM_ENABLE_ZLIB OFF CACHE BOOL "")
```
Comment (vgvassilev) [inline interpreter/CMakeLists.txt]: I'd add a comment explaining why we do not want the zlib from LLVM.

**Comment by oshadura** [comment]
@vgvassilev  I will add test case to roottest!

**Comment by Axel-Naumann** [comment]
IIUC this will disable compression for the PCH - that's not really a viable option...

**Comment by oshadura** [comment]
@Axel-Naumann it is true, I measured 15% of difference of size of PCH,
so I am switching to other ideas how to solve issue ;)

**Comment by vgvassilev** [comment]
That's PR is superseded by PR #1189 .

**Pull request closed without merging**